Source/GitReleaseManager.Cli/Program.cs

```diff
@@ -51,6 +53,28 @@ private static int Main(string[] args)
                   errs => 1);
         }
 
+        private static void CreateFiglet(BaseSubOptions options)
+        {
+            if (options.NoLogo)
+            {
+                return;
+            }
+
+            var version = Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;

```

Comment:
Once released, the InformationVersion will include all information, including the full commit sha and branch name.  Is that what the end user wants, or should this be limited to the MajorMinorPatch?

Reply:
I think outputting the full information would be too long for most cases.
This is why I got the if statement on the line below to only output everything before the `+` sign. IE only outputs MajorMinorPatch on stable releases.

Reply:
Doh! I totally missed the if statement :smile: I stopped reading once I saw the usage of the InformationVersion.  This makes perfect sense to me.